eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -195,6 +206,16 @@ abstract class SafeMojo extends AbstractMojo {
         () -> this.scope
     );
 
+    /**
+     * External tojos.
+     * @checkstyle MemberNameCheck (7 lines)
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    protected final ForeignTojos extTojos = new ForeignTojos(
```

Comment:
@maxonfjvipon Can we use the full name `extrenalTojos` here? `extTojos` could have different meaning/

Reply:
@volodya-lombrozo yes, we can